Gopkg.toml

```diff
@@ -13,6 +13,11 @@
 [[constraint]]
   name = "github.com/joho/godotenv"
   version = "1.3.0"
+
 [[constraint]]
   name = "gopkg.in/DATA-DOG/go-sqlmock.v1"
   version = "1.3.2"
+
+[[constraint]]
+  name = "github.com/satori/go.uuid"
```

Comment:
Use the Google UUID checker

Reply:
looks like its just left over in the toml file

Reply:
Yes this needs to be removed. Where is [google/uuid](https://github.com/google/uuid) though @austincunningham?